### ISSUE TITLE
r5_dockerize controller: loses ip address

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#454]
**Version:** r5_dockerize 50bedbf
**Environment:** dmaster

Overnight, the controller from dmaster, deployed with r5_dockerize, lost its ip address.

Additionally, xcat services not running.

```
root@controller(unknown*) ~]# tabdump osimage
Unable to open socket connection to xcatd daemon on localhost:3001.
Verify that the xcatd daemon is running and that your SSL setup is correct.
Connection failure: IO::Socket::INET: connect: Connection refused at /opt/xcat/lib/perl/xCAT/Client.pm line 217.
[root@controller(unknown*) ~]# service xcatd status
cat: /var/run/xcatd.pid: No such file or directory
xCAT service is not running
```

**Comment by themisa** [comment]
same today, Friday Sept:
[root@dmaster ~]# ssh -Y node001
ssh: connect to host node001 port 22: No route to host

**Comment by themisa** [comment]
Services (containers) not up after rpower.

```
[root@controller(unknown*) ~]# service xcatd status
cat: /var/run/xcatd.pid: No such file or directory
xCAT service is not running
[root@controller(unknown*) ~]# docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
[root@controller(unknown*) ~]# docker images
REPOSITORY                        TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
controller:5050/nova_controller   latest              808f8911effd        16 hours ago        372.9 MB
controller:5050/glance            latest              af5c21a9ec4f        16 hours ago        343.3 MB
controller:5050/keystone          latest              e41bc6aa6c80        16 hours ago        333.5 MB
controller:5050/rabbitmq          latest              de3cd06ac3bd        16 hours ago        248.7 MB
controller:5050/mariadb           latest              a44ad3e2ac4a        16 hours ago        361.8 MB
controller:5050/trinity           latest              b0a16566d94e        16 hours ago        703.1 MB
docker.io/centos                  latest              0f73ae75014f        2 weeks ago         172.3 MB
docker.io/centos                  centos7             0f73ae75014f        2 weeks ago         172.3 MB
```

**Comment by abhimukh** [comment]
The problem is that dhclient is not running on the controller.

```
[root@controller(unknown*) ~]# systemctl status dhclient
dhclient.service
   Loaded: not-found (Reason: No such file or directory)
   Active: inactive (dead)
```

**Comment by themisa** [comment]
Fixed in r6 ea67ae8
